packages/portfolio-contract/src/portfolio.flows.ts

```diff
@@ -83,16 +160,89 @@ export const openPortfolio = (async (
   ctx: {
     zoeTools: GuestInterface<ZoeTools>;
     makePortfolioKit: () => PortfolioKit;
+    contractAddresses: {
+      aavePoolAddress: string;
+      compoundAddress: string;
+      factoryAddress: string;
+    };
     inertSubscriber: GuestInterface<ResolvedPublicTopic<never>['subscriber']>;
   },
   seat: ZCFSeat,
-  _offerArgs: unknown, // TODO: USDN/USDC ratio
+  offerArgs: {
+    evmChain?: SupportedDestinationChains;
+  }, // TODO: USDN/USDC ratio
   // passed as a promise to alleviate contract start-up sync constraints
   localP: Promise<OrchestrationAccount<{ chainId: 'agoric-any' }>>,
 ) => {
   await null; // see https://github.com/Agoric/agoric-sdk/wiki/No-Nested-Await
   try {
-    const kit = ctx.makePortfolioKit();
+    const { makePortfolioKit, contractAddresses } = ctx;
+    const kit = makePortfolioKit();
+
+    const initRemoteEVMAccount = async () => {
+      const { evmChain } = offerArgs;
+      assert(evmChain, 'evmChain is required to open a remote EVM account');
+
+      const [agoric, axelar] = await Promise.all([
+        orch.getChain('agoric'),
+        orch.getChain('axelar'),
+      ]);
+
+      const { chainId, stakingTokens } = await axelar.getChainInfo();
+      const remoteDenom = stakingTokens[0].denom;
```

Comment:
What if `stakingTokens` is empty? If not, do we know that the `[0]`th entry in the array is the right one?

If we know at design time that there's exactly 1 staking token for axelar, we can do:

```suggestion
      assert.equal(stakingTokens.length, 1, 'axelar has 1 staking token);
      const [{ denom: remoteDenom }] = stakingTokens;
```